code/src/sixsq/nuvla/server/resources/module.clj

```diff
@@ -168,6 +168,17 @@ component, or application.
       second
       first))
 
+
+(defn count-children-as-admin [parent-path]
+  (-> (crud/query-as-admin
```

Comment:
Running the check as admin to make sure no children are missed: makes sense ?

Reply:
Yes.